internal/sshserver/dist/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## Changelog
```

Comment:
This changelog seems to be unnecessary.

Reply:
This is autogenerated from goreleaser, was added accidentally. Whoops.